code/game/objects/items/weapons/vaurca_items.dm

```diff
@@ -0,0 +1,189 @@
+			user << "<span class='notice'>After a few seconds of remembering, you input the solution to the riddle - a lovely riddle indeed - and open the box to reveal an ancient thing.</span>"
+			var/obj/item/weapon/archaeological_find/X = new /obj/item/weapon/archaeological_find
+			user.remove_from_mob(src)
+			user.put_in_hands(X)
+			qdel(src)
+
+	else
+		user << "<span class='notice'>You stare at the box for a few seconds, trying to even comprehend what you're looking at... (You must stand still to complete the puzzle box.)</span>"
+		if(do_after(user, 60))
+			user << "<span class = 'notice'>After a few more seconds, you hesitantly turn the first piece of the puzzle box.</span>"
+			if(do_after(user,30))
+				user << "<span class = 'notice'>After nothing bad happens, you dive into the puzzle with a feeling of confidence!</span>"
+				if(do_after(user,200))
+					user << "<span class = 'notice'>Twenty seconds pass, and suddenly you're feeling a lot less confident. You struggle on...</span>"
+					if(do_after(user,100))
+						user << "<span class='notice'>You blink, and suddenly you've lost your place, your thoughts, your mind...</span>"
+						if(do_after(user,20))
+							user << "<span class='notice'>You find yourself again, and get back to turning pieces. At this point it is just randomly.</span>"
+							if(do_after(user,600))
+								user << "<span class='notice'>A minute goes by, and with one final turn the box looks just like it did when you started. Fucking bugs.</span>"
+
+/obj/item/weapon/melee/vaurca/navcomp
+	name = "navcomp coordinate archive"
+	desc = "A rather heavy data disk for a Vaurcae Arkship navigation drive."
+	icon = 'icons/obj/vaurca_items.dmi'
+	icon_state = "harddisk"
+	force = 10
+	throwforce = 5
+	w_class = 3
+	contained_sprite = 1
+
+/obj/item/weapon/melee/vaurca/rock
+	name = "Sedantis rock"
+	desc = "A large chunk of alien earth from the distant Vaurcae world of Sedantis I. Just looking at it makes you feel funny."
+	icon_state = "glowing"
+	icon = 'icons/obj/vaurca_items.dmi'
+	force = 15
+	throwforce = 30
+	w_class = 4
+	contained_sprite = 1
+
+/obj/item/weapon/grenade/spawnergrenade/vaurca
+	name = "K'ois delivery pod"
+	desc = "A sophisticated K'ois delivery pod, for seeding a planet from the comfort of space."
+	spawner_type = /obj/machinery/portable_atmospherics/hydroponics/soil/invisible
+	deliveryamt = 7
+	contained_sprite = 1
+	icon = 'icons/obj/vaurca_items.dmi'
+	icon_state = "beacon"
+	force = 15
+	throwforce = 30
+	w_class = 4
+
+/obj/item/weapon/grenade/spawnergrenade/vaurca/prime()
+
+	if(spawner_type && deliveryamt)
+		var/turf/T = get_turf(src)
+		playsound(T, 'sound/effects/phasein.ogg', 100, 1)
+		for(var/mob/living/carbon/human/M in viewers(T, null))
+			if(M.eyecheck() < FLASH_PROTECTION_MODERATE)
+				flick("e_flash", M.flash)
+
+		for(var/i=1, i<=deliveryamt, i++)
+			var/obj/machinery/portable_atmospherics/hydroponics/soil/invisible/x = new spawner_type(T,/datum/seed/koisspore)
```

Comment:
This is why you needed the `new newseed` directive. You were sending a _path_. Proper version is this:

```
		for(var/i=1, i<=deliveryamt, i++)
			var/obj/machinery/portable_atmospherics/hydroponics/soil/invisible/x = new spawner_type(T, new /datum/seed/koisspore())
```

Or if you enjoy clarity and very robust wording.
```
		for(var/i=1, i<=deliveryamt, i++)
			var/datum/seed/koisspore/KS = new()
			var/obj/machinery/portable_atmospherics/hydroponics/soil/invisible/x = new spawner_type(T, KS)
```